### PR TITLE
[WIP] 20181122 fixes

### PR DESCRIPTION
**Pull request by enginehouse**

### CODE REVIEW

File: app/shared/components/Pagination/component.jsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ export default class PaginationBoxView extends React.PureComponent {
 
     const previousClasses = previousClassName + (current === 0 ? ` ${disabledClassName}` : '')
     const nextClasses = nextClassName + (current === pageCount - 1 ? ` ${disabledClassName}` : '')
-
+    console.log(current, (pageCount - 1))
     return (
```
Comment (openbook) [inline app/shared/components/Pagination/component.jsx]: CONSOLE CONSLOE CNSLE

File: app/shared/utilities.js [hunk 2]
```diff
@@ -84,21 +84,29 @@ export function exists (obj, key) {
...
-  if (path.imageMedium && path.imageMedium.fields) {
+  if (path.imageMedium) {
     imageObj[path.mediumBreakpoint] = path.imageMedium.fields.file.url || 600
   }
 
-  if (path.imageSmall && path.imageSmall.fields) {
+  if (path.imageSmall) {
     imageObj[path.smallBreakpoint] = path.imageSmall.fields.file.url || 400
   }
   return imageObj
```
Comment (openbook) [inline app/shared/utilities.js]: I put these (  && path.imageSmall.fields  ...   && path.imageMedium.fields) to catch when contentful sledge hammers them out of the system (and causes the page to die), so worth reinstating them.

**Comment by enginehouse** [comment]
@openbook - would it be ok to re-review this please?

**Comment by openbook** [comment]
tres tres bon

**Pull request merged**